Esterni/Piano_di_progetto/res/sections/consuntivi/progettazione_dettaglio.tex

```diff
@@ -1,11 +1,48 @@
 \subsubsection{Periodo complessivo}
 
-	\subsubsubsection{Prospetto orario}
-
-	\subsubsubsection{Prospetto economico}
-
+	\subsubsubsection{Consuntivo}
+		\begin{longtable}{
+			>{\centering}p{0.25\textwidth}
+			>{\centering}p{0.08\textwidth}
+			>{\centering}p{0.08\textwidth}
+			>{\centering}p{0.15\textwidth}
+			>{\centering\arraybackslash}p{0.15\textwidth} }
+		
+		\coloredTableHead
+		\textbf{\color{white}Ruolo} &
+		\textbf{\color{white}Ore} &
+		\textbf{\color{white}Delta ore} &
+		\textbf{\color{white}Costo in \euro{}} &
+		\textbf{\color{white}Delta costo}
+		\tabularnewline
+		\endhead
+		
+		% Contenuto della tabella
+		% Ruolo & OreEffettive & DeltaOre & Costo & DeltaCosto \\
+		Responsabile    & 23 & +0 &   690,00 & +  0,00 \\
+		Amministratore  & 26 & -6 &   520,00 & -  120,00 \\
+		Analista        & 3 & +3 &   75,00 & + 75,00 \\
+		Progettista     & 94 & +6 & 2068,00 & + 132,00 \\
+		Programmatore   & 145 & -8 &   2175,00 &  -120,00 \\
+		Verificatore    & 98 & +2 & 1470,00 & + 30,00 \\
+		\textbf{Totale Effettivo} & \multicolumn{2}{c}{\textbf{389}} & \multicolumn{2}{c}{\textbf{6998,00}} \\
+		\textbf{Delta} & \multicolumn{2}{c}{\textbf{-3}} & \multicolumn{2}{c}{\textbf{-3,00}} \\
+		
+		\rowcolor{white}\caption{Consuntivo per il periodo di Progettazione di Dettaglio e Codifica}	\\
+		
+	\end{longtable}
+	
 	\subsubsubsection{Conclusioni}
-
-	\subsubsubsection{Preventivo a finire}
+	Il bilancio finale è \textbf{positivo}, con un risparmio di \textbf{17,00\euro}. \\ Come riportato nella tabella del consuntivo per il periodo di Progettazione di Dettaglio e Codifica, il numero di ore preventivate per il ruolo di Responsabile si è dimostrato adeguato. \\ Segue un'analisi delle motivazioni per cui gli altri ruoli hanno necessitato di un numero di ore differente da quanto previsto: 
```

Comment:
Qui il risparmio non dovrebbe essere di 3€ come da tabella? (in realtá io scriverei che si é concluso in pari, senza risparmio o spesa in +, dato che sono solo 3€)

Reply:
Sisi scusa, dovrebbero essere 3. Per quanto riguarda il commento, nicoletta proponeva di mantenere la frase. Casomai potremo mettere che il bilancio è positivo (perchè effettivamente abbiamo uno scarto), ma che è irrisorio o cose del genere

Reply:
Sono d'accordo 👍

Reply:
ok allora adesso risolvo